docs/components/best-practices/security/csrf-protection.md

```diff
@@ -0,0 +1,115 @@
+---
+title: CSRF protection configuration
+tags:
+  - CSRF
+  - Security
+  - HTTP
+  - Headers
+  - OWASP
+  - Webapps
+  - Web applications
+description: "Configure CSRF protection for your Camunda instance."
+---
+
+Cross-Site Request Forgery (CSRF) is a type of malicious exploit where unauthorized commands are
+transmitted from a user that the web application trusts. In a CSRF attack, an attacker tricks a victim's
+browser into making unwanted requests to a web application where the victim is authenticated.
+
+For a comprehensive understanding of CSRF attacks and prevention methods, refer to the
+[MDN Web Docs on CSRF](https://developer.mozilla.org/en-US/docs/Glossary/CSRF).
+
+## How CSRF protection works in Camunda
+
+- **Token generation**: A unique CSRF token is generated and stored in a secure, HTTP-only cookie named `X-CSRF-TOKEN`.
+- **Token validation**: For state-changing requests (POST, PUT, DELETE, etc.), the server validates that the CSRF token
+  in the request header matches the one in the cookie.
+- **Safe methods**: GET, HEAD, TRACE, and OPTIONS requests are considered safe and don't require CSRF validation.
+
+## Configuration
+
+### Enabling/disabling CSRF protection
+
+CSRF protection is enabled by default for security. You can configure it using:
+
+**Application property:**  
+`camunda.security.csrf.enabled: true  # Set to 'false' to disable CSRF protection`
+
+**Environment variable:**  
+`CAMUNDA_SECURITY_CSRF_ENABLED=true  # Set to false to disable CSRF protection`
+
+:::caution
+Disabling CSRF protection is not recommended for production environments as it leaves your application vulnerable to cross-site request forgery attacks.
+:::
+
+## Authentication method differences
```

Comment:
Are there really particular differences here? The bullet points in the lists are pretty similar. If we want to keep this, we should also talk about the case of deactivated authentication.